ex6.3_full_names.py

```diff
@@ -0,0 +1,23 @@
+def full_names(first_names_list, last_names_list, min_length=None):
```

Comment:
Use type hints and return types -docum
https://www.geeksforgeeks.org/duck-typing-in-python/
https://stackoverflow.com/questions/125367/dynamic-type-languages-versus-static-type-languages
https://docs.python.org/3/library/typing.html
https://realpython.com/lessons/pros-and-cons-type-hints/